manifests/init.pp

```diff
@@ -61,7 +61,16 @@
   Optional[Integer] $logconfig_file_max_log_size = undef,
   Optional[Integer] $logconfig_file_backups      = undef,
   Optional[String] $logconfig_file_category      = undef,
-) {
+
+  # Padoptions
+  Optional[Etherpad::Stpadoptions] $padoptions = undef,
```

Comment:
I wonder if this can be `Etherpad::Stpadoptions $padoptions = {},` which would remove the need for the if statement below.